client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.js

```diff
@@ -12,7 +12,10 @@ angular.module('managerApp').config(($stateProvider) => {
       },
     },
     translations: {
-      value: ['../../../billing/vouchers/addCredit'],
+      value: [
```

Comment:
There is `noTranslations: true` right above, is it still relevant ? What if you remove it ?

Reply:
'billing/vouchers/addCredit' translations are used in this page. I am not sure why noTranslations is set to true. Its legacy code, I did not change it.